### DISCUSSION TITLE
Vertical Menu - Super link of sub links does not work

### DISCUSSION THREAD

**Discussion by srimarru** [Azure/api-management-developer-portal#2167] [General]
![image](https://user-images.githubusercontent.com/3597676/235801427-8af1d94b-d404-49a5-91e1-31904be6323e.png)

When a vertical menu is created in the Navigation in the Api portal, and links are as children to a given parent link, a parent link is not working. i.e parent link non clickable!  I think with an intension of making the menu items toggle an attribute called data-toggle="dropdown" is added to the a href html element. But that has made the link non functioning! The code related to vertical menu is part of  \node_modules\@paperbits\core\menu.

**Comment by azaslonov** [comment]
@srimarru, you're right, that's bug. We'll include the fix into next release.

**Comment by srimarru** [comment]
Thank you @azaslonov
